### ISSUE TITLE
Unable to reach my arduino files in my USB drive (on Ubuntu)

### ISSUE THREAD

**Issue by SkyGC** [snapcrafters/arduino#17]
hi.
I have an issue with the app.
When i want to open a project on my usb drive, Linux doesn't let me use the Arduino IDE instead of the notepad to open the file that I am looking for (see the first screenshot we can see that the Arduino IDE program doesn't appear (even if i scroll down the list))
![Capture d’écran de 2021-02-13 10-21-41](https://user-images.githubusercontent.com/79002067/107846622-6a1a9f00-6de5-11eb-8850-513b7edf1c65.png)
![Capture d’écran de 2021-02-13 10-24-29](https://user-images.githubusercontent.com/79002067/107846672-c8e01880-6de5-11eb-8beb-8cdf0e5c8124.png)
![Capture d’écran de 2021-02-13 10-24-47](https://user-images.githubusercontent.com/79002067/107846675-ced5f980-6de5-11eb-9768-9ab18c37e8b5.png)
![Capture d’écran de 2021-02-13 10-25-01](https://user-images.githubusercontent.com/79002067/107846677-d4334400-6de5-11eb-946e-f863f8c80dd4.png)
(that's all the list that i have)

 and when I am using the Arduino IDE, I can't access to my usb drive to save the .ino file into it (see the second screenshot).
![Capture d’écran de 2021-02-13 10-26-44](https://user-images.githubusercontent.com/79002067/107846706-ff1d9800-6de5-11eb-926c-b5e0fb0739d5.png)
(as you can see, my usb drive doesn't appear)

Does someone have the same problem than me ?
Does someone knows how to fix it ?

(sorry if you can't understand my message. I'm french and i need a few english improvements )

**Comment by merlijn-sebrechts** [comment]
This is indeed an issue, but you can easily work around it. The USB drive should be in "Ordinateur" > "media" > "your username" > "Name of USB drive".

**Comment by SkyGC** [comment]
Thanks.
This worked fine.
Now i can code without problems ^^!

**Comment by merlijn-sebrechts** [comment]
Reopening this since it is indeed a bug, even though we have a workaround

**Comment by Monmoy042** [comment]
Is it okay to run "sudo usermod -a -G dialout kms" this command in my home directory??

**Comment by merlijn-sebrechts** [comment]
@Monmoy042 

I'm not entirely sure what you're asking. You can run this command wherever you like. It will add the user `kms` to the group `dailout`. This should give the user `kms` access to Arduino boards connected over USB tty.

**Comment by Monmoy042** [comment]
@galgalesh yes it works perfectly in my system. I forgot to mention you that it worked in my system.
